result/lldp.yaml

```diff
@@ -0,0 +1,90 @@
+openapi: 3.1.0
+
+info:
+  title: LLDP results models
+  version: ^0.0.0
+
+components:
+  schemas:
+    Lldp.Metrics.Request:
+      description: >-
+        The request to retrieve LLDP per interface metrics/statistics.
+      type: object
+      properties:
+        lldp_names:
+          description: >-
+            The names of LLDP interfaces to return results for.
+            An empty list will return results for all LLDP interfaces.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - "/components/schemas/Device.Lldp/properties/name"
+        column_names:
+          description: >-
+            The requested list of column names for the result set.
+            If the list is empty then metrics for all columns will be returned.
+            The name of LLDP interface can not be excluded.
+          type: array
+          items:
+            type: string
+            enum:
+              - frames_in
```

Comment:
I think we mostly use tx/rx for in/out packets.

Reply:
changed to tx and rx